src/components/Amount/Amount.tsx

```diff
@@ -65,7 +66,7 @@ const Crypto = ({
     number: { toCrypto },
   } = useLocaleFormatter()
 
-  const crypto = toCrypto(value, symbol, {
+  const crypto = toCrypto(bnOrZero(value).toFixed(), symbol, {
```

Comment:
q: `numberToCrypto` already does `bnOrZero(num).decimalPlaces(maximumFractionDigits).toFormat(formatOptions)` on the number, so wondering if it would make sense to pass a BN instance and use `BigNumber.Value` (i.e a string, number, or `BigNumber.Instance`) as a type instead of `NumberValue` for crypto, so we don't duplicate the fixed point logic and risk rugging precision? 

https://github.com/shapeshift/web/blob/9e7bfdc31931681eb2060e63650cfb793c7af80c/src/hooks/useLocaleFormatter/useLocaleFormatter.ts#L192-L211

https://github.com/shapeshift/web/blob/9e7bfdc31931681eb2060e63650cfb793c7af80c/src/hooks/useLocaleFormatter/useLocaleFormatter.ts#L9

Reply:
Actually make sense, I was also wondering about adding explicit `undefined` to the value type and remove the bnOrZero logic from `Amount.Crypto` but I feel like I prefer to force devs to push defined value into `toCrypto`

Modified in [f3858c4](https://github.com/shapeshift/web/pull/7310/commits/f3858c42f4de7a3e5367270f1ef802c1d2257692)